Task 2/2.2/Barriers.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _2._2
+{
+    public class Tree : Barrier
+    {
+        public Tree(int x, int y) : base('T', x, y)
+        {
+            this.x = x;
+            this.y = y;
+        }
```

Comment:
You are sending position to base ctor **and** set it here too - no need, base ctor already sets it.